keyboards/beekeeb/3w6hs/rules.mk

```diff
@@ -0,0 +1,19 @@
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
+UNICODE_ENABLE = yes
+CUSTOM_MATRIX = lite
+NO_USB_STARTUP_CHECK = yes
+LTO_ENABLE = no
+
+SRC += matrix.c
+QUANTUM_LIB_SRC += i2c_master.c
```

Comment:
```suggestion
I2C_DRIVER_REQUIRED = yes
```

Reply:
@drashna  When I apply this change,  I got ```3w6hs/matrix.c:63: undefined reference to `i2c_writeReg'```.

Reply:
That is because your repo is out of date.
https://docs.qmk.fm/#/ChangeLog/20231126?id=peripheral-subsystem-enabling

Updating your repo will fix that.